float64 heave
float64 heaveConf
float64 yaw
float64 yawConf
